data/conf/rspamd/local.d/composites.conf

```diff
@@ -76,7 +76,7 @@ ENCRYPTED_CHAT {
 CLAMD_SPAM_FOUND {
   expression = "CLAM_SECI_SPAM & !MAILCOW_WHITE";
   description = "Probably Spam, Securite Spam Flag set through ClamAV";
-  score = 5;
+  score = 1;
```

Comment:
Lowering it is ok, but i would lower it to 2 instead of 1.

Reply:
I by myself checked how that db works, honestly dropped it immediately. It is not worth a score of 2, and honestly it is not worth even a score of 1. This db has conditions like "jr@gmail.com", which will fire on anything, like abramsjr@gmail.com, anythingjr@gmail.comunity... They not even put a space or <> in their db even they could which will lower FP, they could do this DB in YARA format which would not be so FP as YARA allows nice regexes, but no..., it's one total FP, and they are not responding to email to fix issues in their DB or any FP cases they put there.

Reply:
On every site this spammarketing db listed as highly false positive and not recommended for a use :), Google about it a bit